akka-actor-tests/src/test/java/akka/dispatch/JavaFutureTests.java

```diff
@@ -51,7 +51,7 @@ public String apply(String s) {
       }
     }, system.dispatcher());
 
-    assertEquals("Hello World", Await.result(f2, timeout));
+    assertEquals("Hello World", Await.result(f2, fiveSecondsTimeout));
```

Comment:
I'm not sure this is helpful - it makes the assertions (somewhat) harder to read and it's not really relevant how long the timeout is...